enforcer-rules/src/test/java/org/apache/maven/enforcer/rules/files/TestRequireFilesDontExist.java

```diff
@@ -107,15 +106,58 @@ void testEmptyFileListAllowNull() {
     @Test
     void testFileDoesNotExist() throws EnforcerRuleException, IOException {
```

Comment:
I don't understand this test. It checks that we create a file, delete it, and then that the file doesn't exist? Consider renamingi

Reply:
This seems to just be a minor variation on the original test code, but I can change the name.
What do you think "testDeletedFileDetected"?
I'll update the symbolic link est to follow a similar formate.

Reply:
yes, renaming is a good idea

Reply:
Please give me a format of the new method name you'd prefer and I will ensure that all `test...Exists` and `test...DoesNotExist` methods are updated in all test classes.
Such a change will affect `TestRequireFilesSize`, `TestRequireFileChecksum` `TestRequireFilesDontExist`, and `TestRequireFilesExist`; this means touching classes that are unrelated to the bug fix in order to ensure naming consistency.

To be clear, I disagree with this but it is not my project and so I will follow your direction.

Reply:
Don't change other methods not related to this PR. keep the PR focused.

Reply:
Better 9 wrong and 1 right than 10 wrong. That's the fundamental principle.

Reply:
testSymbolicLinkDeletedDetected (spelling) but otherwise sounds good

Reply:
Done